### DISCUSSION TITLE
Welcome to a9_libraries Discussions!

### DISCUSSION THREAD

**Discussion by tejpochiraju** [IoTReady/a9_gsm_gps_library#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by flaborg16** [comment]
Hi I have a question. Is it possible to use GPIO pins to connect to other sensors and Is it possible to use I2C pins to connect MPU6050 accelerometer? Please help me out.

Thank you,
Mahipal

**Comment by tejpochiraju** [comment]
@flaborg16 - yes it is possible. However, we are no longer maintaining this library as we don't use it. @KaranRajPradhan may have some references on how to do this.
